dev-debian/Dockerfile

```diff
@@ -23,7 +23,8 @@ ARG SIA_DIR="/sia"
 ARG SIA_DATA_DIR="/sia-data"
 ARG SIAD_DATA_DIR="/sia-data"
 
-RUN apt-get update && apt-get install -y mime-support logrotate
+RUN apt-get update && apt-get install -y mime-support logrotate cron && \
+    mv /etc/cron.daily/logrotate /etc/cron.hourly
```

Comment:
I don't think we need to run it hourly, daily should be fine right?

Reply:
Once a day is mostly OK but with some log spam we might get burnt.